src/Docfx.App/Config/BuildJsonConfig.cs

```diff
@@ -13,7 +13,7 @@ namespace Docfx;
 /// BuildJsonConfig.
 /// </summary>
 /// <see href="https://dotnet.github.io/docfx/reference/docfx-json-reference.html#12-properties-for-build"/>
-internal class BuildJsonConfig
```

Comment:
This change seems no behavior differences.
Access to this class from `docfx` is already allowed by `InternalsVisibleTo` attribute.

Reply:
Right but it helps https://github.com/dotnet/docfx/pull/9986 too and makes this PR more "readable" since it does not need to check the `InternalsVisibleTo` so think it is worth it to include it - but if blocking I can remove it, trying to explain the rational.

Reply:
Default class visibility is `internal` when declared under namespace.
It thought it need to specify `public` for consistency with PR comments. (or wait #8872 is resolved)

https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers#class-and-struct-accessibility

> Classes and structs declared directly within a namespace (aren't nested within other classes or structs) can be either public or internal. internal is the default if no access modifier is specified.